packages/compiler/core/scanner.ts

```diff
@@ -1092,6 +1115,34 @@ export function createScanner(
     return (token = Token.Identifier);
   }
 
+  function scanBacktickedIdentifier(): Token.Identifier {
```

Comment:
Just to confirm, do we want backtick strings to follow normal escaping rules (minus backtick handling)? It seems suspicious since backticks are more identifier like and less string-like. You could imagine scanning literally everything until the closing tick, and only allowing escpaing of backticks... not arguing for it necessarily, just want to make sure we think about it.

Reply:
My thinking was we wanted parity with string escaping so we could always format a backcompat quoted member name to a backticked member name. In the edge case of `"\n"` we wouldn't be able to use backticks to pretty print it if we didn't allow escaping other than backtick.

I would propose the following:

Allow `\`` inside quotes too and remove the special case to only allow it in backticks. This would keep the quotes and backticks consistent in the simplest way possible. The other consistent way would be to disallow `\"` in backticks but that feels unnecessarily complicated.

Reply:
+1, support this proposal

Reply:
(It is also humorous that this is impossible to talk about in markdown, lol, tried and failed in teams too)

Reply:
(I think I fixed it above but I supsect this will haunt me forever, lol.)

Reply:
> Allow \\` inside quotes too and remove the special case to only allow it in backticks.

Sounds good, will update accordingly.

Reply:
Done, now quotes and backticks have the same escape characters